docker-compose.yaml

```diff
@@ -44,7 +44,7 @@ services:
     container_name: postgres-archive
     environment:
       POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: password
+      POSTGRES_PASSWORD: ${DATABASE_PASSWORD:-null}
```

Comment:
If we already have an separate variable for `CACHE_DATABASE_PASSWORD`, should there be a variable for `ARCHIVE_DATABASE_PASSWORD`?